cli/unstable_prompt_select_test.ts

```diff
@@ -0,0 +1,359 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+import { assertEquals } from "../assert/equals.ts";
+import { promptSelect } from "./unstable_prompt_select.ts";
+
+const encoder = new TextEncoder();
+const decoder = new TextDecoder();
+
+function spyFn<T, K extends keyof T>(
```

Comment:
Why create original spy util here? What was wrong with `spy` in `@std/testing/mock`?

Reply:
`spy` didn't work for that purpose, but `@std/testing/mock`'s `stub()` did it 👍